### PR TITLE
chore: remove old IE and Edge icons

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

File: src/pages/_document.tsx [hunk 1]
```diff
@@ -147,7 +147,6 @@ export async function renderPage(
                 <link rel="icon" href="/favicon-16x16.png" sizes="16x16" type="image/png">
                 <link rel="manifest" href="/site.webmanifest">
                 <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#33aa33">
-                <meta name="msapplication-TileColor" content="#333333">
```
Comment (styfle) [inline src/pages/_document.tsx:150]: Isn't this used for pinning to desktop on Windows, not necessary just Edge

https://stackoverflow.com/q/61686919/266535

Comment (XhmikosR) [reply src/pages/_document.tsx:150]: The link you shared only mentions Windows 8 and IE, both of which are no more, though.

Comment (styfle) [reply src/pages/_document.tsx:150]: image

Comment (styfle) [reply src/pages/_document.tsx:150]: Did you try it on the latest windows?

Comment (XhmikosR) [reply src/pages/_document.tsx:150]: I don't see any difference because it only affected Windows 8/8.1 and or IE which are dead :)

image

**Pull request merged**